From 3a589dedfbf2b0ada111a3e2cb6b14837f2c8b5e Mon Sep 17 00:00:00 2001
From: Hans Breuer <hans@breuer.org>
Date: Sat, 06 Jun 2009 18:01:38 +0000
Subject: Bug #585004 - implicit declaration of function 'finite' (2nd iteration)

To get the definition either _BSD_SOURCE or _SVID_SOURCE needs to
be defined before inclusion of <math.h>.
Removed HAVE_CONFIG_H and chanegd inclusion order
(cherry picked from commit 8f363f6dd82f9fbf502fceaf1d73a258c17593db)
---
diff --git a/objects/standard/arc.c b/objects/standard/arc.c
index 28302a6..3a50838 100644
--- a/objects/standard/arc.c
+++ b/objects/standard/arc.c
@@ -16,13 +16,11 @@
  * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
  */
 
-#ifdef HAVE_CONFIG_H
 #include <config.h>
-#endif
 
-#include <assert.h>
 #define _BSD_SOURCE 1 /* to get finite */
 #include <math.h>
+#include <assert.h>
 
 #include "intl.h"
 #include "object.h"
--
cgit v0.9.0.2
